.github/workflows/release.yml

```diff
@@ -120,7 +120,7 @@ jobs:
         uses: actions-rs/cargo@v1.0.3
         with:
           command: wix
-          args: -v --nocapture -I install/windows/main.wxs --target ${{ matrix.target }} --output target/wix/starship.msi
+          args: -v --no-build --nocapture -I install/windows/main.wxs --target ${{ matrix.target }} --output target/wix/starship-${{ matrix.target }}.msi
```

Comment:
Is the addition of `--no-build` intentional here?

Reply:
Yes, it's just for a tiny speed-up – If you prefer, I could remove it again.

Reply:
No problem! Just making sure it was intentional.